src/dfinstancewindows.cpp

```diff
@@ -217,7 +217,7 @@ bool DFInstanceWindows::set_pid(){
         return false;
 
     do {
-        if (!_tcscmp(pe32.szExeFile, _T("Dwarf Fortress.exe")))
+        if (!wcscmp(pe32.szExeFile, L"Dwarf Fortress.exe"))
```

Comment:
Just curious, was it causing issues?

It is a good change for consistency with the use of wide characters everywhere else anyway.

Reply:
It caused a compiler error, when compiling for cygwin. It seems that the cygwin headers have some quirks here. I also think that for consistency reasons you should stick to one kind of string functions.  In this case this also solved the problem when compiling for cygwin.